Class-Examples/XML/emilyAssignment3.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<root>
+    <para num="1">Knowing how to change a tire is a necessary skill for all drivers. If you rely on
+        a cell phone to save you in a roadside emergency, there’s always that chance you will forget
+        to charge it, be out of range, or leave it at home. Flat tires can happen anywhere, and a
+        cell phone is no substitute for knowing how to change a flat tire.</para>
+    <para num="2">Thankfully, changing a tire isn’t all that hard! Just adhere to the following
+        guidelines to be prepared in case you have a flat.</para>
+    <para num="3">
+        <heading>ITEMS YOU'LL NEED TO FIX A FLAT TIRE</heading>
+        <subheading>These items should have come with your vehicle:</subheading>
+        <item xml:id="jack">Jack</item>
```

Comment:
@epw5194 I merged this file into main days ago in class, which means I never left you comments on it! I wanted to show you how **line-commenting** works in GitHub. You can open a link to any commit you make to a GitHub repo, and click on the + at the start of a line to craete a little pop-up window to write comments like this. Mostly we do line-commenting on Pull Requests as a way to leave specific comments and suggestions about code. Here, I wanted to highlight that I like how you structured your markup to feature the items in these tire repair instructions. I especially like how you set up `@xml:id` attributes. I wanted to follow up to explain how you might **make use** of those distinct identifiers in those xml:ids. ...